### ISSUE TITLE
Request: Command warmups

### ISSUE THREAD

**Issue by TomLewis** [EssentialsX/Essentials#1320]
We have command cooldowns, can we please get command warmups

**Comment by mdcfe** [comment]
Could you elaborate? I don't know what you mean - delays before executing commands? Having to wait for a certain length of time after connecting to run a command?

 To me, it seems like the different scenarios for triggering these would be used by only a few people.

**Comment by TomLewis** [comment]
Yes a warm up is a delay before the command is executed. For example essentials has this already when you send a teleport request and have to stand still for 5 seconds. This behavior but for commands we set.

**Comment by robertsmrek** [comment]
I would want to help with this @mdcfe

**Comment by JasonHorkles** [comment]
Make a PR then

**Comment by robertsmrek** [comment]
@mdcfe please, check my pull request

**Comment by JasonHorkles** [comment]
The team will review it when they have time

**Comment by robertsmrek** [comment]
Hello @mdcfe, 
just wanted to kindly ask if you could take a look at my PR when you get a chance.
I’d really appreciate your feedback so we can move the feature forward.

Thanks!

**Comment by robertsmrek** [comment]
Hi @JasonHorkles @mdcfe ,
just following up on my PR when you have a moment. Please let me know if there’s anything I should update or adjust. Thanks!
